css/layout-css/news-feed-style.upload.css

```
@@ -1564,6 +1564,9 @@ html.ie11 .new-news-feed-list-container .news-feed-list-item:not(.open) .slide-o
 
   .news-feed-detail-wrapper .news-feed-list-detail-content-scroll-wrapper {
     position: relative;
+  }
+
+  .news-feed-detail-overlay .news-feed-detail-wrapper .news-feed-list-detail-image-wrapper ~ .news-feed-list-detail-content-scroll-wrapper .news-feed-item-inner-content {
```

Selector seems unnecessarily long. Same problem with other layouts.

```suggestion
  .news-feed-list-detail-image-wrapper ~ .news-feed-list-detail-content-scroll-wrapper .news-feed-item-inner-content {
```

---

I see. This is fine then 👍🏻